tools/scripts/ansible/cldr-apps-playbook.yml

```diff
@@ -38,6 +38,8 @@
         mode: "0644"
       notify: Restart Tomcat
     - name: Checkout CLDR trunk
+      become: yes
+      become_user: surveytool
```

Comment:
This is working -- we get an incomplete git repository, and owner is tomcat -- I suspect that instead of become_user, we could wait until after checkout, then use file module to change ownership of CLDR trunk to surveytool